lms/views/basic_lti_launch.py

```diff
@@ -133,7 +118,7 @@ def db_configured_basic_lti_launch(self):
             self.request.db, tool_consumer_instance_guid, resource_link_id
         )
 
-        self._set_via_url(document_url)
+        self.context.js_config.add_document_url(document_url)
```

Comment:
These views are getting perilously close to all having the same code bar a parameterised `document_url`. We should probably revisit when this is all over to see if there's a common function that could be extracted.